classes.py

```diff
@@ -1 +1,38 @@
-# some_code
+def gc(self):
+    gc_count = self.sequence.count('C') + self.sequence.count('G')
+    gc_content = gc_count * 100 / len(self.sequence)
+    return round(gc_content, 2)
+
+
+class Dna:
+    sequence = ''
```

Comment:
I think, that this line is unnecessary. If you want to say, that input data is a string, you can inherit your class from a string.

Reply:
Can you add initialization of sequence in def __init__?